ballerina/natives.bal

```diff
@@ -116,17 +116,19 @@ public  class Counter {
     # + desc - Description of the Counter instance. If no description is provided, the the default empty string
     #          will be used
     # + tags - The key/value pair of Tags. If no tags are provided, the default nil value will be used
-    public function init(string name, string? desc = "", map<string>? tags = ()) {
+    public isolated function init(string name, string? desc = "", map<string>? tags = ()) {
         self.name = name;
         if (desc is string) {
             self.description = desc;
         } else {
             self.description = "";
         }
         if (tags is map<string>) {
-            self.metricTags = tags;
+            self.metricTags = tags.cloneReadOnly();
         } else {
-            self.metricTags = DEFAULT_TAGS;
+            lock {
+	            self.metricTags = defaultTags.cloneReadOnly();
```

Comment:
Formatting is off?

Reply:
Checked and seems like the formatting is correct. Anyway these locks have been removed